changelogs/changelog.yaml

```diff
@@ -2304,3 +2305,22 @@ releases:
     - ec2_vpc_route_table_info-filter-fix.yml
     - release_summary.yml
     release_date: '2023-08-03'
+  6.4.0:
+    changes:
+      minor_changes:
+      - ec2_key - add support for new parameter ``file_name`` to save private key
+        in when new key is created by AWS. When this option is provided the generated
+        private key will be removed from the module return (https://github.com/ansible-collections/amazon.aws/pull/1704).
+      release_summary: This release brings a new module named ``amazon.aws.ec2_key_info``,
+        some documentation improvements and a security fix for the ``amazon.aws.ec2_key``
+        module.
```

Comment:
The ec2_key change only has a 'minor_changes' section this might cause confusion if we describe it in the release summary as a security fix.